project/JPMSPlugin.scala

```diff
@@ -51,7 +56,19 @@ object JPMSPlugin extends AutoPlugin {
 
   override lazy val projectSettings: Seq[Setting[_]] = Seq(
     addModules := Seq.empty,
-    modulePath := Seq.empty,
+    moduleDependencies := Seq.empty,
+    // modulePath is set based on moduleDependencies
+    modulePath := {
```

Comment:
Do we still need to expose `modulePath`? Shouldn't that be hidden as an internal detail of `JPMSPlugin`?

Reply:
Unfortunately yes. There are still some places that directly access `modulePath`, like https://github.com/enso-org/enso/blob/906442fe62b42d32546ea4281a3a11870571a75c/build.sbt#L1307.